### PR TITLE
CIRC-6832 - IRONdb 0.19.27 Release Notes

### PR DESCRIPTION
**Pull request by pamaddox**

### CODE REVIEW

File: content/irondb/release-notes.md [hunk 1]
```diff
@@ -5,6 +5,18 @@ weight: 40
 
 # Release Notes
 
+## Changes in 0.19.27
+
+2021-07-27
+
+ * Allow nodes to continue to operate if corruption is detected on NNTBS data shards - log
+   that there has been corruption to the shard to the errorlog and continue operating.
```
Comment (esproul) [inline content/irondb/release-notes.md:13]: We might want to copy from the Inside release notes for the additional info about what logs to look for and what to do if they appear.

Comment (pamaddox) [reply content/irondb/release-notes.md:13]: OK. Where are those located?

Comment (pamaddox) [reply content/irondb/release-notes.md:13]: Nevermind, find them

File: content/irondb/release-notes.md [hunk 2]
```diff
@@ -5,6 +5,26 @@ weight: 40
...
+      ```
+      Error committing txn: <return code> (error description)
+      Error putting to lmdb cursor: <return code> (error description)
+      ```
+      If these logs appear, contact Circonus Support for help in remediating the
+      issue.
+ * Fix crashes when trying to use graphite find on a node that is not participating in any
+   topology.
+ * Fix memory leaks when fetching graphite data.
+ * fix races that could possibly lead to data corruption in rare cases.
```
Comment (esproul) [inline content/irondb/release-notes.md:25]:
```suggestion
 * Fix races that could lead to data corruption in rare cases.
```

**Review by esproul — CHANGES_REQUESTED** [review]
Small nit.

**Review by esproul — APPROVED** [review]

**Pull request merged**